### DISCUSSION TITLE
New fields

### DISCUSSION THREAD

**Discussion by LfJohnVo** [coderflexx/laravel-ticket#22] [Q&A]
HI!!!! can i add new fields like project id o resolver? :D

**Comment by ousid** [comment]
Yes, you can. By extending the `Ticket` Model
```php
class Ticket extends \Coderflexx\LaravelTicket\Models\Ticket
{
...
}
```

Then create the new columns via `artisan make:migration` command.

Extending the `Ticket` model, is optional, if you know what you are doing.
